tests/concepts/t3330.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-description: "explain reporting isn't working during the transition to nkError; revise and ressurrect once fixed"
+description: "From https://github.com/nim-lang/Nim/issues/3330"
```

Comment:
I guess that's better than before. 😆

Reply:
didn't have the courage to rename due to the implications on changing nimout. This is passing, btw